app/assets/stylesheets/app/calendar.scss

```diff
@@ -19,8 +19,9 @@
     }
 
     .day.current-month:hover {
-      background-color: $secondary !important;
+      background-color: $primary !important;
       color: $light;
+      cursor: pointer;
```

Comment:
to let the user know they can click to add dates. chose the link color to emphasize clicking.